app/models/apple/episode.rb

```diff
@@ -491,7 +491,7 @@ def audio_asset_state_success?
     end
 
     def has_media_version?
-      return false unless delivery_status.present? && delivery_status.source_media_version_id.present?
+      return false unless delivery_status.source_media_version_id.present?
```

Comment:
Hrm... could `delivery_status` be nil?  Looks like that aliases to a has_many.first on the Episode.  Maybe a `&.` safe nav?

Reply:
This was one of the changes in this PR, to set up a default status.

So when querying the status, the default is set up:
https://github.com/PRX/feeder.prx.org/pull/1116/files#diff-4ced10ac3e0d2a50f776b961b3f61a72a3ba92bf4dd67664942cdefe0b72f2f4R28-R33

and when updating the status, the default is provided:
https://github.com/PRX/feeder.prx.org/pull/1116/files#diff-932681036f4ca05556eefe956f92cff8ad8bc433cda926d65d22a701b8c5f050R6

The idea is that we can move methods from the episode model over onto this status model (provided we initialize a default).  Ultimately, this lets us use instance methods over on the status model (like `def increment_asset_wait`).